src/main/java/world/bentobox/bentobox/managers/SchemsManager.java

```diff
@@ -74,7 +71,7 @@ private void copySchems(Addon addon, File folder) {
      * @return map of schems for this world or an empty map if there are none registered
      */
     public Map<String, Clipboard> get(World world) {
-        return islandSchems.getOrDefault(world, new HashMap<>());
+        return islandSchems.getOrDefault(world, new TreeMap<>(String.CASE_INSENSITIVE_ORDER));
```

Comment:
What does it bring to use a TreeMap there?

Reply:
It is not possible to use insensitive order with standard hashmap.

Reply:
Or you can just put String#toLowerCase() or String#toUperCase().
There will never be more than 100 elements, so even List or NoDuplicationList is possible can be used.

TreeMap definitely saves memory... but ... not so many elements will be there.

Reply:
@BONNe names in the map should'nt be case insensitive, because the schem file won't load then. Now map stores case-sensitive keys and when player writes insensitive string, it returns schem with case-sensitive name, so schem with case-sensitive name will be loaded.